### PR TITLE
esti run job for testin with kv enabled

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: .github/workflows/esti.yaml [hunk 1]
```diff
@@ -721,6 +721,68 @@ jobs:
...
+      - name: Start lakeFS with S3 tests KV
+        uses: ./.github/actions/bootstrap-test-lakefs
+        with:
+          compose-file: esti/ops/docker-compose.yaml
+          compose-flags: "--quiet-pull --exit-code-from=esti"
+        env:
+          LAKEFS_BLOCKSTORE_TYPE: s3
+          LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID: ${{ secrets.ESTI_AWS_ACCESS_KEY_ID }}
+          LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY: ${{ secrets.ESTI_AWS_SECRET_ACCESS_KEY }}
+          LAKEFS_DATABASE_KV_ENABLED: true
```
Comment (itaiad200) [inline .github/workflows/esti.yaml:751]: Dumb question but learning from graveler migration: How would we know that the KV was actually used?

Comment (nopcoder) [reply .github/workflows/esti.yaml]: It includes the log warning that we currently "USING KV EXPERIMENTAL FLAG!!! USE AT YOUR OWN RISK!!!"

File: .github/workflows/esti.yaml [hunk 2]
```diff
@@ -721,8 +721,73 @@ jobs:
...
+        uses: ./.github/actions/bootstrap-test-lakefs
+        with:
+          compose-file: esti/ops/docker-compose.yaml
+          compose-flags: "--quiet-pull --exit-code-from=esti"
+        env:
+          LAKEFS_BLOCKSTORE_TYPE: s3
+          LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID: ${{ secrets.ESTI_AWS_ACCESS_KEY_ID }}
+          LAKEFS_BLOCKSTORE_S3_CREDENTIALS_SECRET_ACCESS_KEY: ${{ secrets.ESTI_AWS_SECRET_ACCESS_KEY }}
+          LAKEFS_DATABASE_KV_ENABLED: "true"
+          LAKEFS_DATABASE_TYPE: postgres
```
Comment (itaiad200) [inline .github/workflows/esti.yaml:752]: Where's the DSN?

**Review by itaiad200 — CHANGES_REQUESTED** [review]
Waiting for a check that KV table exists in postgres DB

File: esti/system_test.go [hunk 3]
```diff
@@ -220,3 +221,29 @@ func listRepositories(t *testing.T, ctx context.Context) []api.Repository {
 	}
 	return listedRepos
 }
+
+// TestKVEnabled tests that lakefs database contains kv table in case feature is enabled
+func TestKVEnabled(t *testing.T) {
```
Comment (itaiad200) [inline esti/system_test.go:226]: How do you know that it's not running first and then the KV table is empty?

**Review by itaiad200 — APPROVED** [review]
Thanks for adding the check

**Pull request merged**